pyinjective/composer.py

```diff
@@ -460,7 +435,7 @@ def MsgResponses(data, simulation=False):
             "/cosmos.bank.v1beta1.MsgSend": cosmos_bank_tx_pb.MsgSendResponse,
             "/cosmos.authz.v1beta1.MsgGrant": cosmos_authz_tx_pb.MsgGrant,
```

Comment:
This should be cosmos_authz_tx_pb.MsgGrantResponse and similarly all the below, we're looking for the response rather than the request.

Reply:
got it